### ISSUE TITLE
Illegal Access Warning When Compiling

### ISSUE THREAD

**Issue by mtchd** [mtchd/connect4#1]
![giphy (1)](https://user-images.githubusercontent.com/30946820/62999351-81f02a80-beb1-11e9-9fa8-52c0fbdda638.gif)

See below:

```
WARNING: An illegal reflective access operation has occurred
WARNING: Illegal reflective access by com.google.protobuf.UnsafeUtil (file:/***/scala-2.12.7/org.scala-sbt/sbt/1.2.6/protobuf-java-3.3.1.jar) to field java.nio.Buffer.address
WARNING: Please consider reporting this to the maintainers of com.google.protobuf.UnsafeUtil
WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
WARNING: All illegal access operations will be denied in a future release
```

**Comment by mtchd** [comment]
Potential solution: Update protobuf library.

**Comment by mtchd** [comment]
Fixed in 0.2.1 by forcing the the program to load a later version of the protobuf library.
